src/typescript-service.ts

```diff
@@ -497,7 +489,8 @@ export class TypeScriptService {
 			span.log({ event: 'fetched package files' });
 
 			// Search symbol in configuration
-			const config = this.projectManager.getConfiguration(packageRoot);
+			// forcing TypeScript mode
+			const config = this.projectManager.getConfiguration(packageRoot + 'file.ts');
```

Comment:
this feels hacky. I would add an optional parameter `type` to `getConfiguration()` that specifies the type and defaults to `this.getConfigurationType(fileName)`